### PR TITLE
HIP 84: Correct links and grammatical clarifications

### PR DESCRIPTION
**Pull request by Siegfried-B**
A few very small suggestions 
- to correct an inconsistent phrasing
- to improve readability by shortening sentences without affecting their meaning.

### CODE REVIEW

**Comment by abhay** [comment]
Although these are good grammatical improvements, I'm loathe to merge these changes at this time given the vote is live.

**Comment by Siegfried-B** [comment]
Great!
Do we have a policy how to handle HIPs that have been voted on? There should be.
On the one hand, I think they shouldn't be touched.
On the other hand, correcting typos, missing words and the like that don't touch the content one bit should be possible.
In any case, the version that has been voted upon should be archived and very easy to find and access.

**Comment by abhay** [comment]
These changes are so innocuous that I'm okay with the grammatical updates. In my mind, HIPs are supposed to update docs once they're implemented.

**Comment by abhay** [comment]
Made a small edit and link to approved version for reference, @Siegfried-B. Will merge if you're good with that change.

File: 0084-service-provider-hex-boosting.md [hunk 1]
```diff
@@ -3,13 +3,14 @@
 - Author(s): Mobile PoC Working Group
 - Start Date: 2023-04-26
 - Category: Economic, Technical
-- Original HIP PR: #625
-- Tracking Issue: #638
+- Original HIP PR: [#625](https://github.com/helium/HIP/pull/625)
+- Tracking Issue: [#638](https://github.com/helium/HIP/issue/638)
```
Comment (abhay) [inline 0084-service-provider-hex-boosting.md:7]:
```suggestion
- Tracking Issue: [#638](https://github.com/helium/HIP/issues/638)
```

**Comment by Siegfried-B** [comment]
I'm good with merging (didn't check the added URLs, I trust they're correct).

**Pull request merged**